spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -3,18 +3,93 @@
 RSpec.describe Api::V1::KeywordsController, type: :request do
   describe 'GET#index' do
     it 'returns success status' do
-      get api_v1_keywords_path
+      header, = create_token_header
+
+      get api_v1_keywords_path, headers: header
 
       expect(response).to have_http_status(:success)
     end
 
     it 'returns the valid keywords' do
-      get api_v1_keywords_path
+      header, = create_token_header
+
+      get api_v1_keywords_path, headers: header
 
       keywords = JSON.parse(response.body, symbolize_names: true)[:data]
 
       expect(keywords[0][:attributes][:name]).to eq('First keyword')
       expect(keywords[1][:attributes][:name]).to eq('Second keyword')
     end
   end
+
+  describe 'POST#index' do
+    context 'when csv file is valid' do
```

Comment:
```suggestion
    context 'when CSV file is valid' do
```

same for the many other locations Comma Separated Values is used.